lib/attrsets.nix

```diff
@@ -1,5 +1,5 @@
+/* Operations on attribute sets. */
 { lib }:
-# Operations on attribute sets.
```

Comment:
Why doesn't nixdoc support `#` as a comment character? A design choice?

Reply:
Yeah, I think this was used to be able to distinguish between doc comments and other comments. By using `/*` for all doc comments it's clear what's gonna get rendered.

I guess this won't be a concern anymore once [RFC 145](https://github.com/NixOS/rfcs/pull/145) is implemented in nixdoc though :)